iree/tools/init_compiler_modules.h

```diff
@@ -22,16 +22,22 @@
 namespace mlir {
 namespace iree_compiler {
 
+// Add all the IREE compiler module dialects to the provided registry.
+inline void registerIreeCompilerModuleDialects(DialectRegistry &registry) {
+  // clang-format off
```

Comment:
Why `clang-format off`?

Reply:
This is taken over from upstreamÖ https://github.com/llvm/llvm-project/blob/05777ab941063192b9ccb1775358a83a2700ccc1/mlir/include/mlir/InitAllDialects.h#L41